MonoGame.Framework/Graphics/Texture2D.cs

```diff
@@ -27,28 +29,58 @@ public Rectangle Bounds
 				return new Rectangle(0, 0, this.width, this.height);
             }
         }
-
+        /// <summary>
+        /// Creates a new texture of the given size
+        /// </summary>
+        /// <param name="graphicsDevice"></param>
+        /// <param name="width"></param>
+        /// <param name="height"></param>
         public Texture2D(GraphicsDevice graphicsDevice, int width, int height)
             : this(graphicsDevice, width, height, false, SurfaceFormat.Color, SurfaceType.Texture, false, 1)
         {
         }
-
+        /// <summary>
+        /// Creates a new texture of a given size with a surface format and optional mipmaps 

```

Comment:
@Tzenchor - In the future remember to put periods at the end of the sentence in the reference docs.  I didn't get time to review this PR before it was merged else I would have caught this then.